docs/maintain-guides-society.md

```diff
@@ -0,0 +1,159 @@
+
+![Society Dashboard](assets/society/submit_bid.jpg)
+
+Anyone who wants to join the society that is required to deposit 10 KSM for reserve on Kusama and placing the bid(1 in this case) that you want to get when joining the society. 
+
+![Society Dashboard](assets/society/test_bid.jpg)
+
+Once you have submitted the transaction, your bid will be shown on the [Society page](https://polkadot.js.org/apps/#/society) under the bids section. You can cancel the bidding if you decided not to join the society by calling the `unbid(pos)`.   
+
+```bash
+unbid(pos)
+
+pos - The index of the bid. 
+```
+
+If you are not sure what your position is, you can check that by going to the [Society->bids](https://polkadot.js.org/apps/#/chainstate) the PolkadotJS apps.
+
+You can find an existing member by placing a bid on your behalf if you do not have KSM and you are willing to give a tip to them. An existing member can submit a `vouch(who,value,tip)` transaction.
+
+```bash
+vouch(who,value,tip)
+
+who - Which user you are vouching for
+value - The value that the user would like to get when joining the society
+tip - Fees you get
+
+Note: The final value that the candidate will get = (value - tip)
+```
+
+![Society Dashboard](assets/society/vouch.jpg)
+
+
+### 2. Candidate Phase
+
+![Society Vote Candidate](assets/society/vote_candidate.jpg)
+
+
+```bash
+vote(candidate,approve)
+
+candidate - Select the candidate you would like to vote
+approve - Yes / No
+```
+
+Those selected bids in this phase will be voted by the existing members to decide whether you will be approved to join the society. Members will vote for all the candidates and the final outcome will be randomly selected by one of the votes. Let's take a look the example shown as below:
+
+```bash
+Note:
+- If the randomly selected member does not vote, it will be treated as a rejection.
+- For each rotation period, the maximum number of members that can be accepted is set as 10. 
+```
+
+A - Accept,  R - Reject,  S - Skeptic
+
+Member | 1 | 2 | 3 | 4 | 5 |
+--- | --- | --- | --- | ---- | --- |
+Vote | A | A | A | R | S |
+Selected | | | X | | | 
+
+In this example, this candidate will be approved to join the society since member 3 was selected as a final outcome. Also, a number of members will be randomly chosen as `skeptics` to vote for the candidates during the rotation period. Since members 5 was chosen as a skeptic, he is required to participate in the voting process. If he does not participate in voting, he will be punished by increase his strike per missing vote. And if you have accumulated too many stikes, your membership will be suspended which means you may need to re-apply the membership and your unclaimed payouts will be slashed. Moreover, each member who voted opposite to the randomly selected vote will be slashed their unclaimed payouts and increase their strikes. In this case, member 4 will be punished.
```

Comment:
```suggestion
In this example, a candidate will be approved to join the society since member 3 was selected as a final voting outcome. A number of members will also be randomly chosen as "skeptics" to vote for the candidates during the rotation period. Since member 5 was chosen as a skeptic, they are required to participate in the voting process. If they do not participate in voting, they will be punished with one strike per missing vote. If one accumulates too many strikes, one's membership is suspended which means they may need to re-apply and their unclaimed payouts will be slashed. Moreover, each member who voted opposite to the randomly selected vote will have their unclaimed payouts slashed and strikes increased. In this case, member 4 will be punished.
```